### DISCUSSION TITLE
Confusion between RawG and Raw

### DISCUSSION THREAD

**Discussion by jamesleeht** [aarondl/sqlboiler#1150] [Q&A] [answered]
Based on the source code here: https://github.com/volatiletech/sqlboiler/blob/master/queries/query.go

It seems like both Raw and RawG do the same thing and don't require any executor. Is there still a reason to use RawG?

**Comment by stephenafamo** [comment, marked as answer]
You're correct. Perhaps there was a difference at some point in the past and now it is just there for backward compatibility.

Personally, I hadn't noticed it at all. It should probably be removed in the next major version.
